### PR TITLE
added sample code for batch_query usage

### PR DESCRIPTION
**Pull request by eyvindn**

### CODE REVIEW

File: api/client/samples/batch_queries/batch_queries.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    'metric_id': 860032, 
+    'item_id': 274, 
+    'source_id': 25, 
+    'frequency_id': 9,
+    'start_date': '1998-01-01T00:00:00.000Z',
+    'end_date': '1998-01-01T00:00:00.000Z'
+}
+
+# make a list of this query for every county in Mississippi
+mississippi_county_ids = api_client.search_and_lookup("regions", "Mississippi").next()["contains"]
```
Comment (nemozen) [inline api/client/samples/batch_queries/batch_queries.py]: get_descendant_regions

File: api/client/samples/batch_queries/batch_queries.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+selections = []
+for region_id in mississippi_county_ids:
+    selection_temp = dict(selection)
+    selection_temp["region_id"] = region_id
+    selections.append(selection_temp)
+
```
Comment (nemozen) [inline api/client/samples/batch_queries/batch_queries.py]: add a main() (see e.g. quick_start.py)

**Review by nemozen — CHANGES_REQUESTED** [review]

File: api/client/samples/batch_queries/batch_queries.py (hunk 2 above)
Comment (eyvindn) [reply api/client/samples/batch_queries/batch_queries.py]: done

File: api/client/samples/batch_queries/batch_queries.py (hunk 1 above)
Comment (eyvindn) [reply api/client/samples/batch_queries/batch_queries.py]: done

**Review by nemozen — APPROVED** [review]

**Pull request merged**